app-server/app/src/main/scala/user/User.scala

```diff
@@ -1,37 +1,66 @@
 package com.azavea.rf.user
 
 import scala.concurrent.{Future, ExecutionContext}
+import java.sql.Timestamp
 
 import com.azavea.rf.datamodel.latest.schema.tables.Users
 import com.azavea.rf.datamodel.latest.schema.tables.UsersRow
 import com.azavea.rf.utils.Database
 
+case class UsersRowCreate(
```

Comment:
I think we should follow from our python style guide -- two lines between package level objects/classes, one line between functions within classes/objects